app/core/tests/test_models.py

```diff
@@ -18,12 +18,12 @@ def test_value_registration():
     with pytest.raises(ValidationError):
 
         @models.register_value_model
-        class InvalidValueModel(models.GenericValue):
+        class InvalidValueModel(models.GenericValue):  # pylint: disable=unused-variable
```

Comment:
should just put these pylint disables at the top for test code

Reply:
Sure! I had thought you wanted pylint to run on the tests
https://github.com/samplchallenges/SAMPL-league/pull/110#discussion_r711263412
But you were just asking why, not saying we shouldn't.